### ISSUE TITLE
Unable to get all resources for a tenant in Azure Storage Explorer

### ISSUE THREAD

**Issue by dhananjayspeaks** [microsoft/AzureStorageExplorer#7745]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Regression From

1.28

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

All

### OS Version

Windows 11

### Bug Description
```
{
  "name": null,
  "message": "\"{\\n  \\\"name\\\": \\\"Error\\\",\\n  \\\"message\\\": \\\"Failed to parse response body: <!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Transitional//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd'><html xmlns='http://www.w3.org/1999/xhtml'><head><meta content='text/html; charset=utf-8' http-equiv='content-type'/><style type='text/css'>body { font-family:Arial; margin-left:40px; }img  { border:0 none; }#content { margin-left: auto; margin-right: auto }#message h2 { font-size: 20px; font-weight: normal; color: #000000; margin: 34px 0px 0px 0px }#message p  { font-size: 13px; color: #000000; margin: 7px 0px 0px 0px }#errorref { font-size: 11px; color: #737373; margin-top: 41px }</style><title>Microsoft</title></head><body><div id='content'><div id='message'><h2>The request is blocked.</h2></div><div id='errorref'><span>Ref A: B55E0315736040AE9256643CD554B68F Ref B: MAA201060516039 Ref C: 2024-02-14T09:25:22Z</span></div></div></body></html>\\\",\\n  \\\"stack\\\": \\\"Error: Failed to parse response body: <!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Transitional//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd'><html xmlns='http://www.w3.org/1999/xhtml'><head><meta content='text/html; charset=utf-8' http-equiv='content-type'/><style type='text/css'>body { font-family:Arial; margin-left:40px; }img  { border:0 none; }#content { margin-left: auto; margin-right: auto }#message h2 { font-size: 20px; font-weight: normal; color: #000000; margin: 34px 0px 0px 0px }#message p  { font-size: 13px; color: #000000; margin: 7px 0px 0px 0px }#errorref { font-size: 11px; color: #737373; margin-top: 41px }</style><title>Microsoft</title></head><body><div id='content'><div id='message'><h2>The request is blocked.</h2></div><div id='errorref'><span>Ref A: B55E0315736040AE9256643CD554B68F Ref B: MAA201060516039 Ref C: 2024-02-14T09:25:22Z</span></div></div></body></html>\\\\n    at sendRequestAndParseResponse (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\AzureSubscriptions\\\\\\\\SubscriptionLister.js:122:15)\\\\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\\\\n    at async SubscriptionLister.listSubscriptions (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\AzureSubscriptions\\\\\\\\SubscriptionLister.js:33:36)\\\\n    at async SubscriptionListingClient.runOperation (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\node_modules\\\\\\\\@storage-explorer\\\\\\\\azure-sdk-helpers\\\\\\\\dist\\\\\\\\src\\\\\\\\CoreRest\\\\\\\\CoreRestClient.js:110:24)\\\\n    at async SubscriptionService._getAccountSubscriptionsForTenants (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\AzureSubscriptions\\\\\\\\SubscriptionService.js:42:49)\\\\n    at async SubscriptionService.getTenantSubscriptions (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\AzureSubscriptions\\\\\\\\SubscriptionService.js:24:16)\\\\n    at async AzureSubscriptionsManager.getAccountSubscriptions (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\AzureSubscriptions\\\\\\\\AzureSubscriptionsManager.js:27:22)\\\\n    at async Provider.executeOperation (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\providers\\\\\\\\Provider.js:28:16)\\\\n    at async ElectronMainHost.executeOperation (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\ElectronMainHost.js:26:20)\\\\n    at async ElectronRendererProviderProxy._handleExecuteRequest (C:\\\\\\\\Program Files\\\\\\\\Microsoft Azure Storage Explorer\\\\\\\\resources\\\\\\\\app\\\\\\\\out\\\\\\\\app\\\\\\\\main\\\\\\\\ElectronRendererProviderProxy.js:145:31)\\\"\\n}\""
}
```
### Steps to Reproduce

1. I singed in with my Azure subscription.
2. I don't see any resource from my tenant. I have only one tenant in Azure.

### Actual Experience

I successfully sing in using Azure option in Azure Storage Explorer. But it's unable to load my subscription and resources for the tenant. I have only one tenant in my Azure. 

### Expected Experience

It should show me all resources attached with the tenant, and I should be able to see my Storage account in Explorer window.

**Comment by craxal** [comment]
Part of the error message says that the "request is blocked". Do you know if you have the necessary permissions to list subscriptions? Are you able to see subscriptions when using the Azure portal?

**Comment by dhananjayspeaks** [comment]
I was using this subscription and resources attached to this subscription. Infact, Azure Storage Explorer, Visual Studio Code from Azure Functions development tools, etc. used to work absolutely fine.

I had Windows 11 update a few days back and then I started getting this error.

⁣Get BlueMail for Android ​

**Comment by dhananjayspeaks** [comment]
I got the solution today. A couple of days ago, my Windows 11 got updates and it did something with the proxy settings. I had been facing problem with all the tools be it Azure Storage Explore or VS Code with Azure Tools, I was getting same error. Then, I tried from another laption with same OS config, and everything worked fine i.e. no error at all. Hence, I was sure it's something with my OS settings.

I just hard booted my Wifi switch and BOOM! Everything started fine. Give it a try before you dig more into Azure tools setttings, etc.
